scripts/package_examples.rb

```diff
@@ -53,11 +50,13 @@ def replace_in_file(filepath, pattern, replacement)
 end
 
 framework_directory_for_example = {
-  'examples/ios/xcode-7/objc' => '../../../../ios/static/xcode-7',
+  'examples/ios/objc' => '../../../ios/static',
   'examples/osx/objc' => '../../../osx',
   'examples/tvos/objc' => '../../../tvos',
   'examples/ios/swift-2.2' => '../../../ios/swift-2.2',
-  'examples/tvos/swift-2.2' => '../../../tvos',
+  'examples/tvos/swift-2.2' => '../../../tvos/swift-2.2',
```

Comment:
Is the lack of 2.3 for these because none of our examples are compatible with 2.3? If there's any non-packaging work needed for them then it seems fine to not bother (for now at least), but it does seem a little strange to be missing them.